mu-plugins/central-hub/tests/phpunit/unit/config-store/_loadConfigFromFilesystem.php

```diff
@@ -45,4 +46,39 @@ public function test_should_return_array_when_path_to_file_is_given() {
 		$this->assertSame( 'foo', $actual[0] );
 		$this->assertSame( $expected, $actual[1] );
 	}
+
+	/**
+	 * Test _load_config_from_filesystem() should throw an Exception when configuration no store key exists.
+	 */
+	public function test_should_throw_exception_when_no_store_key() {
+		// Test when no store key but has configuration parameters.
+		$path_to_file = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/config-with-params-only.php';
+		$this->expectException( \Exception::class );
+		$this->expectExceptionMessage(
+			sprintf( 'No store key exists in the %s configuration file.', $path_to_file )
+		);
+		_load_config_from_filesystem( $path_to_file );
+
+		// Test when no store key or configuration parameters.
+		$path_to_file = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/empty-config.php';
+		$this->expectException( \Exception::class );
+		$this->expectExceptionMessage(
+			sprintf( 'No store key exists in the [%s] configuration file.', $path_to_file )
+		);
+		_load_config_from_filesystem( $path_to_file );
+	}
+
+	/**
+	 * Test _load_config_from_filesystem() should throw an Exception when configuration store key exists and
+	 *  parameters are empty.
+	 */
+	public function test_should_throw_exception_when_config_store_key_exists_and_no_parameters() {
```

Comment:
Let's shorten the method name like we did with the previous test:

```php

/**
 * Test _load_config_from_filesystem() should throw an Exception when the configuration has no 
 * configuration parameters.
 */
public function test_should_throw_exception_when_config_has_no_parameters()
```

Reply:
Fixed with https://github.com/rgadon107/cornerstone/pull/20/commits/618020b287430994ad863bf1f240ade00863c83a